src/Angular.js

```diff
@@ -583,6 +582,9 @@ function isPromiseLike(obj) {
   return obj && isFunction(obj.then);
 }
 
+function isFloat32Array(value) {
+  return toString.call(value) === '[object Float32Array]';
```

Comment:
should be `isTypedArray()`, tbh. Using O.p.toString isn't very reliable with ES6 anymore, but we should be able to test it against a regexp like `/^[object (Uint8(Clamped)?)|(Uint16)|(Uint32)|(Int8)|(Int16)|(Int32)|(Float(32)|(64)Array]$`, I think